manifests/server/config_entry.pp

```diff
@@ -111,10 +114,13 @@
       # We need to force postgresql to stop before updating the port
       # because puppet becomes confused and is unable to manage the
       # service appropriately.
+      $stop_command = ['service', $postgresql::server::service_name, 'stop']
+      $stop_onlyif = ['service', $postgresql::server::service_name, 'status']
+      $stop_unless = "grep 'PGPORT=${shell_escape($value)}' /etc/sysconfig/pgsql/postgresql"
```

Comment:
isn't this one missing the brackets [[ ]]?

Reply:
No, decided to go without as there is no practical need

Reply:
And as a way to clearly show it hasn't been divided

Reply:
The version without a shell:
```suggestion
      $stop_unless = [['grep', "PGPORT=${value}", '/etc/sysconfig/pgsql/postgresql']]
```

Reply:
👍